saleor/graphql/product/filters.py

```diff
@@ -144,13 +166,19 @@ def filter_products_by_attributes_values(qs, queries: T_PRODUCT_FILTER_QUERIES):
     return qs.filter(*filters)
 
 
-def filter_products_by_attributes(qs, filter_values, filter_range_values):
+def filter_products_by_attributes(
+    qs, filter_values, filter_range_values, filter_boolean_values
+):
```

Comment:
Do we have tests for filter attributes by boolean value?

Reply:
We will. Last thing to add 👮